server/models/report.js

```diff
@@ -44,6 +44,11 @@ reportFilterSchema.add({
     promptInstructions: String,
     promptMandatory: Boolean,
     promptTitle: String,
+    prefillEntries: {
+        show: Boolean,
+        limit: Number,
+        unlimited: Boolean,
+    }
 });
```

Comment:
Existing reports and dashboard should be updated using a database migration file. Otherwise all existing filters will have their suggestions disabled when updating to the new version.
Or instead you can change the code so that when these values are undefined, they defaults to the actual behavior (show: true, limit: 15, unlimited: false), which is better as it does not require filling the database with default values everywhere, and it will even work with imported reports/dashboards that do not have these properties.

Reply:
I fix it in the onInit() filter prompt component, thanks for review